Makefile

```diff
@@ -313,7 +313,7 @@ install-rabbitmq-cluster-operator: | $(KUBECONFIG) $(KUBECTL) ## Install RabbitM
 # https://github.com/jetstack/cert-manager/releases
 # ⚠️  You may want to keep this in sync with RABBITMQ_TOPOLOGY_OPERATOR_VERSION
 # In other words, don't upgrade cert-manager to a version that was released AFTER RABBITMQ_TOPOLOGY_OPERATOR_VERSION
```

Comment:
The RABBITMQ_TOPOLOGY_OPERATOR_VERSION is from Oct 2021. Should we also update that to the latest? (1.3.0) Just thinking about the warning above with keeping the versions in-sync